src/parser/index.js

```diff
@@ -0,0 +1,3 @@
+
+module.exports.asyncParser = require('./AsyncParser').AsyncParser;
```

Comment:
I think we might need to rename the parser to something like `scenarioParser` as otherwise people will get confused with github.com/asyncapi/parser-js